docs/installation/debian.md

```diff
@@ -12,113 +13,121 @@ parent = "smn_linux"
 
-### Installation
+ 9. Purge the old repo if it exists.
 
-1. Install Kernel from wheezy-backports
+         $ apt-get purge lxc-docker*
 
-    Add the following line to your `/etc/apt/sources.list`
+ 10. Verify that `apt` is pulling from the right repository.
 
-    `deb http://http.debian.net/debian wheezy-backports main`
+         $ apt-get policy docker.io
 
-    then install the `linux-image-amd64` package (note the use of
-    `-t wheezy-backports`)
+     From now on when you run `apt-get upgrade`, `apt` pulls from the new apt repository.  
 
-        $ sudo apt-get update
-        $ sudo apt-get install -t wheezy-backports linux-image-amd64
+### Prerequisites on Debian Jessie 8.0 (64-bit)
 
-2. Restart your system. This is necessary for Debian to use your new kernel.
+Debian 8 comes with a 3.16.0 Linux kernel. On Debian the Docker package and executable are called ``docker.io` as the a much older KDE3/GNOME2 package already used the name `docker`. The `docker.io` package can be found
+href="https://lists.debian.org/debian-release/2015/03/msg00685.html"
+target="_blank">in the `jessie-backports` repository</a>.
 
-3. Install Docker using the get.docker.com script:
+Before installing Docker,  <a href="http://backports.debian.org/Instructions/"
+target="_blank">enable the jessie-backports` repository</a>.
 
-    `curl -sSL https://get.docker.com/ | sh`
+### Prerequisites Debian Wheezy/Stable 7.x (64-bit)
 
->**Note**: If your company is behind a filtering proxy, you may find that the
->`apt-key`
->command fails for the Docker repo during installation. To work around this,
->add the key directly using the following:
->
->       $ curl -sSL https://get.docker.com/gpg | sudo apt-key add -
+<a href="https://github.com/docker/docker/issues/407" target="_blank">Docker
+requires Kernel 3.8+</a>, while Wheezy ships with Kernel 3.2. Fortunately,
+`wheezy-backports` currently has <a
+href="https://packages.debian.org/search?suite=wheezy-backports&section=all&arch=any&searchon=names&keywords=linux-image-amd64"
+target="_blank">Kernel 3.16</a> which is officially supported by Docker.
 
-### Uninstallation
+1. Install Kernel 3.16 from `wheezy-backports`.
 
-To uninstall the Docker package:
+2. Restart your system.
 
-    $ sudo apt-get purge docker-engine
+   This is necessary for Debian to use your new kernel.
 
-To uninstall the Docker package and dependencies that are no longer needed:
 
-    $ sudo apt-get autoremove --purge docker-engine
+## Install Docker
 
-The above commands will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+Before installing Docker, make sure you have set your `apt` repository correctly as described in the prerequisites.
 
-    $ rm -rf /var/lib/docker
+1. Get the latest Docker package.
+
+        $ sudo apt-get update
+
+2. Install Docker.
+
+        $ sudo apt-get install docker.io
```

Comment:
this should be `docker-engine`, not `docker.io` (docker.io is the Debian maintained package)